src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/PaymentSuccess.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@using System.Web.Mvc.Html
+@using EA.Weee.Web.Controllers
+@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model EA.Weee.Web.Areas.Producer.ViewModels.PaymentResultModel
+@{
+    var title = "You have successfully submitted your producer registration";
```

Comment:
Can't find a wireframe for this view anywhere

Reply:
https://liysak.axshare.com/?id=5f29oz&p=76379_-_confirmation_page__1_&g=1